src/Shared/ProgramArguments.Shared.cs

```diff
@@ -387,6 +392,37 @@ public bool TryGetEntryProjectPaths(ISlnGenLogger logger, out IReadOnlyList<stri
             return result.Count > 0;
         }
 
+        private static IEnumerable<string> ExpandWildcards(IEnumerable<string> paths)
+        {
+#if NETFRAMEWORK
+            return paths;
```

Comment:
Why can't this work for .NET Framework?

Reply:
The globbing library depends on newer underlying runtime.

Reply:
Actually, the issue is that the globbing library needs either .NET 4.61, .NET 4.72, or .NET Standard 2.0. Right now, SlnGen is being built against .NET 4.6 and 4.71 and the globbing library won't tolerate.

So I can either leave the new feature so it only works for .NET Core versions, or I can increase the framework versions that the rest of SlnGen is built with.